### ISSUE TITLE
[BUG] No extrusion when printing - Skr 1.3 5xTMC2205 Chimera dual extruder

### ISSUE THREAD

**Issue by Liger0** [MarlinFirmware/Marlin#16358]
### Bug Description

Having 2 extruders, if I manually extrude from the LCD, they do fine. But if I start a print, the pulley just goes forth and back and no extrusion is ever done.

### My Configurations
5x 2209, skr 1,3, cr10s, Chimera dual extruder
[Marlin.zip](https://github.com/MarlinFirmware/Marlin/files/4006102/Marlin.zip)
Bugfix as of 21/12/2019

### Steps to Reproduce

Start a print

**Expected behavior:** 
Print

**Actual behavior:** 
Extruder goes back and forth of the same amount

#### Additional Information
https://www.youtube.com/watch?v=EpZp1Q9OTT4

**Comment by sjasonsmith** [comment]
Can you uncomment the line in Configuration_adv.h with `MINIMUM_STEPPER_PULSE` and see if that improves things?

**Comment by Liger0** [comment]
I actually uninstalled everything and reinstalled again and I don't have those configs anymore.
I was trying to rebuild the firmware but it doesn't work anymore on latest stable if I use the cr10s preconfig (I used the generic one when I built the one that doesn't extrude). #16361
So it's pretty hard to retry at this point unless I am able to flash the cr10s based one and see if I still have that issue.
